src/django_components/expression.py

```diff
@@ -106,9 +106,16 @@ def render(self, context: Context) -> str:
 
 
 def is_aggregate_key(key: str) -> bool:
+    key = key.strip()
     # NOTE: If we get a key that starts with `:`, like `:class`, we do not split it.
     # This syntax is used by Vue and AlpineJS.
-    return ":" in key and not key.startswith(":")
+    return (
```

Comment:
Turns out that there was a test for `is_aggregate_key`, but that test was not running. So after fixing it, I realized that this function had a few bugs.